docs/orchestrate/sauce-connect.md

```diff
@@ -55,9 +88,8 @@ http_proxy=http://host.sauceconnect.internal:1080
 
 ### Standard Proxy Configuration
 
-In most programing language (Go, Python, Node.js...) the standard proxy environment
-variables are supported, and there is no extra step to configure the 
-Sauce Connect.
+In most programming languages (Go, Python, Node.js...) the standard proxy environment
+variables are supported, and there is no extra step to configure the Sauce Connect.
```

Comment:
the Sauce Connect -> Sauce Connect or Sauce Connect Proxy

Reply:
good call, tnx